docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -176,6 +176,22 @@ For each Trino table which maps to BigQuery view there exists a system table whi
 Given a BigQuery view ``customer_view`` you can send query
 ``SELECT * customer_view$view_definition`` to see the SQL which defines view in BigQuery.
 
+.. _bigquery-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access to data and metadata in
```

Comment:
Please re-add "and write access"

Reply:
See https://github.com/trinodb/trino/pull/8923#discussion_r693093050

Reply:
Done.